folly/portability/Time.h

```diff
@@ -40,7 +40,7 @@
 
 // These aren't generic implementations, so we can only declare them on
 // platforms we support.
-#if !FOLLY_HAVE_CLOCK_GETTIME && (defined(__MACH__) || defined(_WIN32))
+#if !FOLLY_HAVE_CLOCK_GETTIME && (defined(__MACH__) || defined(_WIN32)) && !(defined(TARGET_OS_TV) && TARGET_OS_TV)
```

Comment:
FWIW this is broken on iOS with XCode as well. Needed to ad `FOLLY_HAVE_CLOCK_GETTIME` explicitly in RN ☹️.

Reply:
Could we not add this in `folly-config.h.cmake`?

Reply:
Which part? The iOS RN build doesn’t use CMake.

Reply:
Aha now I see the source of my confusion... I was testing Folly from main with RN from 0.73, the Podfile for that branch is different. I'll test with that, and make changes to this PR. Thanks!!!

Reply:
@NickGerleman thanks for the help with this -- I tested this today, and if the RCT-Folly podspec has `FOLLY_HAVE_CLOCK_GETTIME=1` added, this patch is not needed. Closing the PR.